cli/bufflib/src/protobuffers/buff_grpc.rs

```diff
@@ -0,0 +1,123 @@
+// This file is generated. Do not edit
```

Comment:
i'm assuming this is the generated client, i'm wondering if maybe we should locate it closer to the CLI instead in this lib, i think it might make CI easier